app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -4,7 +4,9 @@ class DradisDatatable {
     this.$paths = this.$table.closest('[data-behavior~=datatable-paths]');
     this.dataTable = null;
     this.defaultColumns = this.$table.data('default-columns') || [];
+    this.initialPageLoad = false;
```

Comment:
Why are we going the inverse path? Shouldn't the default initializer behavior to be: initialPageLoad = true; then we check local storage and flip that to false when checking that the key is really present?

Also, it seems that this behavior is closely related to `setupDefaultColumns`, shouldn't that method be only called from within the `setupLocalStorage` method (when the key isn't found), instead of it being called all the time?